webextensions/api/tabs.json

```
@@ -1243,6 +1243,83 @@
               },
               "safari_ios": "mirror"
             }
+          },
+          "defaultZoomFactor": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": "79"
+                },
+                "firefox": [
+                  {
+                    "version_added": "107"
+                  },
+                  {
+                    "version_added": "45",
+                    "version_removed": "107",
+                    "partial_implementation": true,
+                    "notes": "Will always return 1, regardless of the actual value."
+                  }
```

I think it might be confusing to consider this feature removed. Elsewhere we handle similar situations with a note along the lines of:
```suggestion
                  {
                    "version_added": "45",
                    "notes": "Until Firefox 107, returns 1 regardless of the default zoom factor setting."
                  }
```

---

Thanks for reviewing! Seems like a reasonable change to make. FWIW, my inspiration was the compatibility data for [CSS.supports()](https://developer.mozilla.org/Web/API/CSS/supports#browser_compatibility), is a different approach usually taken for WebExtension compat data or is CSS.supports an anomaly?

---

I balked at the idea of marking it as removed and the [scheme guide isn't forthcoming on a standard] (https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#partial_implementation). However, I've done some further digging and found this example in the web extension data:
```
              "safari": [
                {
                  "version_added": "15.4",
                  "partial_implementation": true,
                  "notes": "Since Safari 15.4, this event is also fired in response to a message from webpages allowed in <a href='https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/externally_connectable'><code>externally_connectable</code></a>."
                },
                {
                  "version_added": "14",
                  "partial_implementation": true,
                  "notes": "This event is only fired in response to a message from an extension's containing app, not webpages nor other extensions."
                }
              ],
```
So, I'm now inclined to think that you were probably along the right lines but I would suggest:
```suggestion
                  {
                    "version_added": "107"
                  },
                  {
                    "version_added": "45",
                    "partial_implementation": true,
                    "notes": "Returns 1, regardless of the default zoom factor setting."
                  }
```

---

I also note that under [`version_removed`](https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#version_removed) that "If the feature has not been removed from the browser, this property is omitted, rather than being set to false." In this case, the feature was never removed rather it was fully implemented in 107.

---

Thanks, your suggestion is perfect!